go/client/cmd_account_lockdown.go

```diff
@@ -0,0 +1,187 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"errors"
+	"fmt"
+	"text/tabwriter"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+type CmdAccountLockdown struct {
+	libkb.Contextified
+	SetLockdownMode *bool
+	History         bool
+	Force           bool
+}
+
+func NewCmdAccountLockdown(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	cmd := &CmdAccountLockdown{
+		Contextified: libkb.NewContextified(g),
+	}
+	flags := []cli.Flag{
+		cli.BoolFlag{
+			Name:  "enable",
+			Usage: "Enable account lockdown mode.",
+		},
+		cli.BoolFlag{
+			Name:  "disable",
+			Usage: "Disable account lockdown mode.",
+		},
+		cli.BoolFlag{
+			Name:  "history",
+			Usage: "Print history of lockdown mode changes.",
+		},
+		cli.BoolFlag{
+			Name:  "force, f",
+			Usage: "Don't prompt.",
+		},
+	}
+	return cli.Command{
+		Name:  "lockdown",
+		Usage: "Manage account lockdown mode",
+		Flags: flags,
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(cmd, "lockdown", c)
+		},
+		Description: `When lockdown mode is enabled for an account, some operations are
+   blocked for website sessions, including (but not limited to):
+       - account delete or reset,
+       - posting signatures,
+       - changing password or e-mail address,
+       - changing profile information or profile picture.
+
+   These actions are still possible using the Keybase client.
+
+   Enabling lockdown mode does not add a link to user's sigchain, it
+   is an extension of Keybase API server access control. Lockdown mode
+   offers protection in case of someone stealing your browser session
+   or account password.`,
```

Comment:
@maxtaco do you want to look at help description for this command? thanks

Reply:
I'd say nuke the last paragraph.  No need to flog ourselves!